ci/lib.sh

```diff
@@ -75,7 +75,7 @@ download_artifact() {
   local tmp_file
   tmp_file="$(mktemp)"
 
-  curl -fsSL "$(get_artifact_url "$artifact_name")" > "$tmp_file"
+  curl -fsSL "$(get_artifact_url "$artifact_name")" >"$tmp_file"
```

Comment:
just a personal preference, I like to use the long forms of the flags to make it clearer what's happening

Reply:
I didn't write this 🤷‍♂️ but agree, I generally like things to be extra clear